custom/icds_reports/reports/cas_reach_data.py

```diff
@@ -55,13 +55,34 @@ def get_data_for_daily_usage(date, filters):
     awc_this_month_data = get_data_for_awc_monthly(current_month, config)
     awc_prev_month_data = get_data_for_awc_monthly(previous_month, config)
 
+    if current_month.year != now_date.year or current_month.month != now_date.month:
```

Comment:
This might be more clear to define:

```
current_month_selected = (current_month.year == now_date.year and current_month.month == now_date.month)
if not current_month_selected: 
       yesterday_date = current_month + relativedelta(months=1) - relativedelta(days=1)
```